deployment/README.md

```diff
@@ -1,11 +1,10 @@
 # Human-Connection Nitro \| Deployment Configuration
 
-We deploy with [kubernetes](https://kubernetes.io/). In order to deploy your own
-network you have to [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-and get a kubernetes cluster.
+There are a couple different ways we have tested to deploy an instance of Human Connection, with [kubernetes](https://kubernetes.io/) and via [Helm](https://helm.sh/docs/). In order to manage your own
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-04T13:01:25Z" title="Tuesday, February 4th 2020, 2:01:25 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,11 +1,10 @@
 # Human-Connection Nitro \| Deployment Configuration
 
-We deploy with [kubernetes](https://kubernetes.io/). In order to deploy your own
-network you have to [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-and get a kubernetes cluster.
+There are a couple different ways we have tested to deploy an instance of Human Connection, with [kubernetes](https://kubernetes.io/) and via [Helm](https://helm.sh/docs/). In order to manage your own
```


I would even argue that we do not support plain kubernetes configurations once we're familiar enough with helm.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-04T13:26:43Z" title="Tuesday, February 4th 2020, 2:26:43 pm +01:00">Feb 4, 2020</time>_

---


yeah, I hear ya! One more dependency that needs to be updated/tested